osu.Game/Screens/Play/PlayerSettings/VisualSettings.cs

```diff
@@ -27,12 +27,18 @@ public VisualSettings()
                 {
                     Text = "Background dim:"
                 },
-                dimSliderBar = new PlayerSliderBar<double>(),
+                dimSliderBar = new PlayerSliderBar<double>
+                {
+                    DisplayAsPercentage = true
```

Comment:
Should `PlayerSliderBar` use percentage as default?

Reply:
Would probably leave as is to be consistent with it's base class.